packages/realm/src/app-services/App.ts

```diff
@@ -157,6 +158,7 @@ export type AppConfiguration = {
 };
 
 /**
+ * @deprecated LocalAppConfiguration is no longer used by Atlas Device Sync. It will be removed in future SDK releases and should not be used.
```

Comment:
No need to mention the symbol in its documentation.

```suggestion
 * @deprecated No longer used by Atlas Device Sync. It will be removed in future SDK releases and should not be used.
```